### DISCUSSION TITLE
Please allow showing a streams viewer count in the split header

### DISCUSSION THREAD

**Discussion by legenis** [Chatterino/chatterino2#4764] [Ideas] [answered]
As a streamer, I have been using this application as a replacement for twitch's streamer console because of your applications respect to CPU usage - thank you BTW. Would be really nice to be able to monitor the viewer count or have an option to turn it on. I know there is a viewer list, but for me I normally just want to see the view count # ... it also usually gives more accurate representation of the viewers and not the twitch bots that show up as chatters :)

**Comment by Mm2PL** [comment, marked as answer]
This already exists, you can enable showing the viewer count in the split header.
![image](https://github.com/Chatterino/chatterino2/assets/25011746/12720c6e-ef70-4909-84fa-8d80242698b0)
